test/rendering/listbox/listbox.js

```diff
@@ -5,6 +5,8 @@
       id: `listbox-${+new Date()}`,
       getListObjectData: async () => (fixture ? fixture.getListObjectData() : getMockData()),
       getLayout: async () => (fixture ? fixture.getLayout() : getListboxLayout()),
+      beginSelections: async () => {},
+      selectListObjectValues: async () => ({ qSucsess: true }),
```

Comment:
```suggestion
      selectListObjectValues: async () => ({ qSuccess: true }),
```

Reply:
that the test still worked mean I overcomplicated it and it should be `async () => true` (also updated)